openfl/component/aggregator/straggler_handling.py

```diff
@@ -140,3 +192,71 @@ def __minimum_collaborators_reported(self, num_collaborators_done) -> bool:
                 False otherwise.
         """
         return num_collaborators_done >= self.minimum_reporting
+
+
+class PercentagePolicy(StragglerHandlingPolicy):
+    """Percentage based Straggler Handling function."""
+
+    def __init__(self, percent_collaborators_needed=1.0, minimum_reporting=1, **kwargs):
+        """Initialize a PercentagePolicy object.
+
+        Args:
+            percent_collaborators_needed (float, optional): The percentage of
+                collaborators needed. Defaults to 1.0.
+            minimum_reporting (int, optional): The minimum number of
+                collaborators that should report. Defaults to 1.
+            **kwargs: Variable length argument list.
+        """
```

Comment:
Class docstring should go near attributes, not in `__init__`. Example:

```python
class PercentagePolicy(StragglerHandlingPolicy):
    """Percentage based Straggler Handling function.

    Args:
            percent_collaborators_needed (float, optional): The percentage of
                collaborators required before concluding the round. Defaults to 1.0.
            minimum_reporting (int, optional): The minimum number of
                collaborators that should report, regardless of percentage threshold. Defaults to 1.
            **kwargs: Variable length argument list (karan: why would a user provide this? 
                where does it get used, please mention like "keyword arguments forwarded to some.module.class").
    """

    def __init__(self, percent_collaborators_needed=1.0, minimum_reporting=1, **kwargs):
        if minimum_reporting <= 0:
            raise ValueError("minimum_reporting must be >0")

        self.percent_collaborators_needed = percent_collaborators_needed
        self.minimum_reporting = minimum_reporting
        self.logger = getLogger(__name__)
```

Reply:
The arguments specified are intended for the constructor rather than the class attributes, so they are correctly placed. I have removed the kwargs as per your suggestion.